translations/pencil_zh_CN.ts

```diff
@@ -1827,7 +1827,7 @@
     <message>
         <location filename="../app/ui/mainwindow2.ui" line="886"/>
         <source>Move Frame Backward</source>
-        <translation>先后移动帧</translation>
+        <translation>向后移动帧</translation>
```

Comment:
Does changing the ts files work or does this need to be changed on Transifex? I was under the impression that it needed be changed on Transifex or it will just be overwritten next time the translations are pulled, but I am not fully familiar with that process.

Reply:
I need to upload this CN.ts file to transifex otherwise it will be overwritten next time as you said.

I will do that.